src/include/sof/debug/telemetry/telemetry.h

```diff
@@ -15,6 +15,12 @@
 /* disables calculating systick_averages */
 #define SOF_PERFORMANCE_MEASUREMENTS
 
+/* to be moved to Zephyr */
+#define WIN3_MBASE DT_REG_ADDR(DT_PHANDLE(DT_NODELABEL(mem_window3), memory))
+#define ADSP_PMW ((volatile uint32_t *) \
+		 (sys_cache_uncached_ptr_get((__sparse_force void __sparse_cache *) \
+				     (WIN3_MBASE + WIN3_OFFSET))))
+
```

Comment:
Hmm, but this is now very Intel specific, should this be called something else than debug/telemtry? debug/intel-telemetry.h?

Reply:
Moved new functionality to separate files. I'm guessing we still want those to be called intel-performance-monitor, and intel-telemetry? Do we want to change the CONFIG_SOF_TELEMETRY and CONFIG_SOF_TELEMETRY_PERFORMANCE_MEASUREMENTS names too?